applications/launchpad/backend/src/docker/mod.rs

```diff
@@ -57,3 +62,90 @@ lazy_static! {
 lazy_static! {
     pub static ref CONTAINERS: RwLock<HashMap<String, ContainerState>> = RwLock::new(HashMap::new());
 }
+
+pub fn tari_blockchain_volume_name(tari_workspace: &str, tari_network: TariNetwork) -> String {
+    format!("{}_{}_volume", tari_workspace, tari_network.lower_case())
+}
+
+pub fn network_name(tari_workspace: &str) -> String {
```

Comment:
Confusingly named function.

This doesn't return the network name, but the key for this workspace's entry in a hashmap.

```suggestion
pub fn network_key(tari_workspace: &str) -> String {
```
```suggestion
pub fn network_name(tari_workspace: &str) -> String {
```